app/(pages)/about-us/page.tsx

```diff
@@ -97,6 +143,17 @@ export default function AboutUs() {
           </Grid>
         </Stack>
       </Sheet>
+      <Sheet>
+        <Typography level='h1' className='graphite' textAlign='center' py={2} sx={{marginBottom: 0}}>
+          Our Team
+        </Typography>
+        <Typography level='body' className='graphite' textAlign='center' pb={7}>
+          Meet the team behind the magic!
+        </Typography>
+        <OurTeamProfiles>
+
+        </OurTeamProfiles>
```

Comment:
You can use one tag <OutTeamProfiles /> instead

Reply:
fixed!